scripts/quests/windurst/Reap_What_You_Sow.lua

```diff
@@ -33,21 +33,14 @@ quest.sections =
 
         [xi.zone.WINDURST_WATERS] =
         {
-            ['Mashuu-Ajuu'] =
-            {
-                onTrigger = function(player, npc)
-                    if player:getFameLevel(xi.quest.fame_area.WINDURST) >= 4 then
-                        return quest:progressEvent(483)
-                    else
-                        return quest:progressEvent(463, 0, xi.items.SOBBING_FUNGUS, xi.items.BAG_OF_HERB_SEEDS)
-                    end
-                end,
-            },
```

Comment:
Instead of omitting event 483, let's move the logic down from the quest_available check. Where if players aren't eligible due to the constraints of the odd behavior, it'll give them event 483.

Moreover we should make 483 -> return quest:event(483) rather than progress event

Reply:
So:
if fame >= 4 and not player:hasCompletedQuest(xi.quest.log_id.WINDURST, xi.quest.id.windurst.LET_SLEEPING_DOGS_LIE)) then
event 483